app/src/main/java/com/example/taskyy/ui/screens/ReminderScreen.kt

```diff
@@ -431,7 +436,9 @@ fun TimePickerDialog(
                     ) { Text("Cancel") }
                     TextButton(
                         onClick = {
-                            timeSelected("${pickedTime.hour}:${pickedTime.minute}")
+                            val hour = pickedTime.hour
+                            val minute = pickedTime.minute
```

Comment:
Why not just pass `pickedTime` here which includes all the information about the time you need? Formatting it as a `$hour:$minute` string just makes extra effort having to convert it back to time values